idd/Energy+.idd.in

```diff
@@ -42533,10 +42533,8 @@ Coil:Cooling:DX:VariableSpeed,
         \type alpha
         \reference CoolingCoilsDXVariableSpeed
    A2,  \field Indoor Air Inlet Node Name
-        \required-field
```

Comment:
Air source integrated heat pump (ASIHP) includes multiple, variable-speed coils, which have the same inlet and output nodes. Our design aims to have the ASIHP object to contain the nodes, and pass them into the “child” coil objects, in order to avoid repeatign the nodes in the coils objects, which are prevented by the EP general logic now. So, we removed required-field of nodes for all variable speed DX coil objects.